app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -15,6 +15,11 @@ attributes :gpg_key_id
 attributes :content_id, :content_view_version_id, :library_instance_id
 attributes :product_type
 attributes :promoted? => :promoted
+attributes :distribution_version
+attributes :distribution_arch
+attributes :distribution_bootable? => :distribution_bootable
```

Comment:
This is what started my question around this function, as it seems like you should just rely on the database attribute instead of mapping a function to this attribute?